README.md

```diff
@@ -4,33 +4,84 @@
 [![Go Report Card][report-card-img]][report-card]
 [![License][license-img]][license]
 
-# harbor-scanner-trivy
+# Harbor Scanner Adapter for Trivy
 
-The Harbor Scanner Adapter for [Trivy][trivy-url].
-See [Pluggable Image Vulnerability Scanning Proposal][image-vulnerability-scanning-proposal] for more details.
+The Harbor Scanner Adapter for [Trivy][trivy-url] is a service that translates the [Harbor][harbor-url] scanning API
+into the Trivy commands and allows Harbor to utilize Trivy for providing vulnerability reports on images stored in
+Harbor registry as part of its vulnerability scan feature.
+
+> See [Pluggable Image Vulnerability Scanning Proposal][image-vulnerability-scanning-proposal] for more details.
 
 ## TOC
 
-- [Deploy to Kubernetes (minikube)](#deploy-to-kubernetes-minikube)
+- [Getting started](#getting-started)
+  - [Installation](#installation)
+- [Testing](#testing)
+- [Deployment](#deployment)
 - [Configuration](#configuration)
 - [Documentation](#documentation)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Getting started
+
+These instructions will get you a copy of the adapter service up and running on your local machine for development
+and testing purposes. See [deployment](#deployment) for notes on how to deploy on a live system.
+
+### Prerequisites
+
+* [Go (version 1.13)](https://golang.org/doc/devel/release.html#go1.13)
+* Docker
+
+### Installation
 
-## Deploy to Kubernetes (minikube)
+Run `make` to build the binary in `./scanner-trivy`:
 
 ```
-$ eval $(minikube docker-env -p harbor)
-$ make container
-$ kubectl apply -f kube/harbor-scanner-trivy.yaml
+make
 ```
 
+To build into a Docker container:
+
+```
+make container
 ```
-kubectl port-forward service/harbor-scanner-trivy 8080:8080 &> /dev/null &
 
-curl -v http://localhost:8080/api/v1/metadata
+## Testing
+
+Run `make test` to run all unit tests:
+
+```
+make test
 ```
 
+Run `make test-integration` to run integration tests:
+
+```
+make test-integration
+```
+
+## Deployment
+
+### Kubernetes
+
+1. Create Kubernetes deployment and service for the scanner adapter:
+   ```
+   $ kubectl apply -f kube/harbor-scanner-trivy.yaml
```

Comment:
Does this assume a locally-built image or will it pull from a registry somewhere?

Reply:
I've clarified that by default it's the latest image published to Docker Hub. For local deployment you have to change it manually with `kubectl set image deployment ...`.

At some point we should have a helm chart with image variable and some default value.